modules/ROOT/pages/exception-filter.adoc

```diff
@@ -4,8 +4,20 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: error handling, exceptions, exception catching, exceptions, filter
 
-*Purpose:* Filters messages according to the type of an exception.
+Purpose: +
+Filters messages according to the type of an exception.
 
-*Minimum Configuration:* The class that indicates the type of the exception.
+Minimum Configuration: +
+The class that indicates the type of the exception.
 
-*Discussion:* An Exception Filter matches on the type of an exception. You supply the class indicating the exception type to the Expected Type property. For example, for a runtime exception type, you might set Expected Type to `java.lang.RuntimeException`. You can enter the class name or browse to it.
\ No newline at end of file
+Discussion: +
+An Exception Filter matches the type of an exception you configure in the *Expected Type* field in the *Global Element* properties in Studio, or in the `expectedType` property in the Configuration XML.
+For example, for a runtime exception type, you set *Expected Type* to `java.lang.RuntimeException`. You can enter the class name or browse to it.
+
+The Exception Filter expects an exception from the previous module in the flow; this means that you need to add it within a Catch Exception Strategy processor. The exception will be passed to the next processor in the flow only if it matches the Exception Filter's configured `expectedType`.
```

Comment:
`from the previous module in the flow` -> What does this mean?

Reply:
I was trying to clarify where the exception comes from, but since the filter is in the exception handler this sounds misleading. May be I can change this sentence to just `The Exception Filter expects an exception; this means that you need to add it within a Catch Exception Strategy processor`
Do you think this is ok?